README.md

```diff
@@ -1,62 +1,57 @@
-:wave:	*New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:	
-
+:wave: *New to our project? Be sure to review the [OpenMRS 3 Frontend Developer Documentation](https://openmrs.github.io/openmrs-esm-core/#/). You may find the [Map of the Project](https://openmrs.github.io/openmrs-esm-core/#/main/map) especially helpful.* :teacher:
 
 ![Node.js CI](https://github.com/openmrs/openmrs-esm-patient-management/workflows/Node.js%20CI/badge.svg)
 
-
 # OpenMRS Patient Management
 
-This is a [Lerna](https://lerna.js.org/) project containing patient management frontend modules which deal with creating, searching, and listing patients. This package handles the following:
+This repository bundles together frontend modules for the OpenMRS SPA. These modules handle concerns relating to registering and editing patients, searching for existing patients, creating and managing patient lists and managing patient queues in an outpatient setting. The microfrontends within this module include:
 
--  [@openmrs/esm-patient-search-app](packages/esm-patient-search-app)
--  [@openmrs/esm-patient-registration-app](packages/esm-patient-registration-app)
--  [@openmrs/esm-patient-list-app](packages/esm-patient-list-app)
--  [@openmrs/esm-outpatient-app](packages/esm-outpatient-app)
+- [Active visits app](packages/esm-active-visits-app/)
+- [Outpatient app](packages/esm-outpatient-app/README.md)
+- [Patient search](packages/esm-patient-search-app)
+- [Patient registration](packages/esm-patient-registration-app)
+- [Patient list](packages/esm-patient-list-app)
 
-## Repository Development
+## Setup
 
+Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
 
-### Prerequisites
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna).
 
-- [Node](https://nodejs.org/en/download)
-- yarn ```sh npm install yarn -g ```
-### Getting started
+To start a dev server for a specific microfrontend, run:
 
-To install and setup the repository once cloned just use the following command
-
-```sh
-npx lerna bootstrap
+```bash
+yarn start --sources 'packages/esm-<insert-package-name>-app'
```

Comment:
This is redundant with `yarn start` since it already includes all the packages as `--sources`. But I would actually advocate changing the `package.json` to work the same way it works in patient-chart, with `start` and `start-all` commands.

Reply:
What would `start` do, i.e. which module would be run as the default? In `openmrs-esm-patient-chart`, it's the `patient-chart-app`.

Reply:
Looks like it'll need a path to an import map to work:

<img width="1319" alt="Screenshot 2022-05-04 at 20 16 10" src="https://user-images.githubusercontent.com/8509731/166743010-095f88e7-f309-47d6-959c-e4946e93af3f.png">